src/hooks/useConfirmationText.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+
+function useConfirmationText() {
+  const location = useLocation<any>();
+  const history = useHistory();
+  const [confirmationText, setIsConfirmationText] = useState('');
```

Comment:
I don't think we need the `Is` in the setter:

```suggestion
  const [confirmationText, setConfirmationText] = useState('');
```